docs/ipam.md

```diff
@@ -18,11 +18,13 @@ Field                  | Type   | Description
 ---------------------- | ------ | -----------
 `max-nodes-in-rack`    | int    | The maximum number of nodes in a rack, excluding "boot" node.
 `node-ipv4-pool`       | string | CIDR IPv4 network for node IP pool.
+`node-ipv4-offset`     | string | Offset for dividing the pool into subnets, expressed in IPv4 notation.  Default is "", equivalent to "0.0.0.0".
```

Comment:
Node IPs will be started by adding this to `node-ipv4-pool`.  Default is "", equivalent to `0.0.0.0`.